libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -264,10 +278,14 @@ class Compiler {
   // output.
   bool generate_debug_info_;
 
+  // Total Number of optimization passes to perform.
+  uint32_t num_opt_passes_;
```

Comment:
Previously it's useful since `EnableOptPass` is just a struct. Now not anymore. Removed.